apps/arm/include/ElbowConfig.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <numeric>
+
+#include "AEAT6012.h"
+#include "ActuatorControllerManager.h"
+#include "Current.h"
+#include "DCMotor.h"
+#include "LookupTable.h"
+#include "OpenLoop.h"
+#include "PID.h"
+#include "Position.h"
+#include "Velocity.h"
+
+namespace Elbow {
+
+namespace Internal {
+
+static Encoder::AEAT6012 encoder({ELBW_ENC_SPI_CLK, ELBW_ENC_SPI_MISO, NC, 0});
+
+static Actuator::DCMotor motor(MTR_PWM_ELBW, MTR_DIR_ELBW, false);
+
+// static Sensor::CurrentSensor currentSensor(ELBW_CRNT_SNS_SPI_CLK, ELBW_CRNT_SNS_SPI_MISO, ELBW_CRNT_SNS_SPI_CS);
+// TODO: Add once current sensor driver works
+
+static PID::PID velPID({1, 0, 0, -1, 1, 0, false, false});
+static PID::PID posPID({1, 0, 0, -1, 1, 0, false, false});
+static PID::PID curPID({1, 0, 0, -1, 1, 0, false, false});
+
+constexpr float ASSUNMACURRENT = 25.263;
```

Comment:
typo

Reply:
Also recommend putting units in variable name and separated with underscores like `ASSUN_MAX_CURRENT_AMPS`

Reply:
can the units thing be a later task pls? i was gonna suggest we use a units lib after SAR anyway

Reply:
Sounds good